__tests__/suggestedActions.js

```diff
@@ -148,4 +148,22 @@ describe('suggested-actions command', async () => {
 
     expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
   }, 60000);
+
+  test('should not show suggested actions not destinated for the user', async () => {
```

Comment:
the test was a lot more simple than i thought! 👍

Reply:
Thanks. Downside, if you want to try out corner cases, you will need to make the integration part (e.g. DLJS) fall at the angle you want. E.g. Corina's offline UI work, she actually need to make a faulty DLJS object to achieve that.